_source/_posts/2022-06-16-oauth-java.adoc

```diff
@@ -0,0 +1,389 @@
+---
+layout: blog_post
+title: "OAuth for Java Developers"
+# title: "OAuth 2.0 for Java Developers"
+# title: "OAuth 2.0 in Java"
+# title: "OAuth 2.1 in Java"
+# title: "OAuth 2.0 on the JVM"
+author: matt-raible
+by: advocate
+communities: [java]
+description: ""
+tags: [java, oauth2]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: awareness
+---
+:page-liquid:
+:toc: macro
+:experimental:
+
+You can use OAuth to secure apps, APIs, and devices. OAuth has become more and more popular, especially as developers are asked knit together hundreds of apps and thousands of users in enterprise environments.
+
+The Java ecosystem is vast and wide with over 10 million developers around the world and an abundance of IDEs, build tools, libraries, and frameworks to make them more productive. The infographic below is recommended as a starting point for Java developers. It's designed to help guide your learning as you go and add your own software components while your experience grows.
+
+Click on the image to zoom in.
+
+image::{% asset_path 'blog/java-oauth2-2022/java-roadmap-preview.jpg' %}[alt=2022 Java Developers Roadmap,link={% asset_path 'blog/java-oauth2-2022/java-roadmap.jpg' %}]
```

Comment:
I personally find this a bit distracting in the post as it takes a focus away a bit, but that could just be me, junior-medior devs might find it useful